aggregate/PUMS/pums_0812_1519_demographics.py

```diff
@@ -33,7 +33,6 @@ def acs_pums_demographics(geography: str, year: str, write_to_internal_review=Fa
         census_PUMS=True,
         demographics_category=True,
     )
-    final.dropna(axis=1, how="all", inplace=True)
```

Comment:
Issue with extra `age_p5pl` being added. Think it's this block of code

```python
if census_PUMS and ind == "LEP":
    col_order.append("age_p5pl")
```
I think this can come out entirely? Along with 

```python 
if not census_PUMS:
    col_order.append(f"{ind_category}_pct_denom")
```

Reply:
I will try take both of those comments and rerun results. I was really confused about that reordering function thanks for catching this.